product_docs/docs/migration_portal/4/known_issues_notes.mdx

```diff
@@ -23,15 +23,15 @@ Some unsupported Oracle objects are removed when Migration Portal assesses your
 
 The following objects are removed during the schema assessment:
 
-- Objects related to `MATERIALIZED VIEWS`. For example, backend `TABLE` or `INDEX` statements created to support `MVIEW`. 
-- Objects related to `Queues`.
-- Objects related to `Nested Tables`.
-- Objects related to `XMLType Tables`.
-- Types that depend on `SYSTEM Schemas`.
-- Indexes related to `PRIMARY KEY` and `UNIQUE` constraints.
-- Unsupported system `GRANT` privileges:
-
-  EDB only supports granting `CREATE DATABASE LINK`, `CREATE PUBLIC DATABASE LINK`, `DROP PUBLIC DATABASE LINK` and `EXEMPT ACCESS POLICY` system privileges. Other `GRANT` statements are not supported and removed from the DDL file. 
+- Objects related to `MATERIALIZED VIEWS`, for example, backend `TABLE` or `INDEX` statements created to support `MVIEW`
+- Objects related to `Queues`
+- Objects related to `Nested Tables`
+- Objects related to `XMLType Tables`
+- Types that depend on `SYSTEM Schemas`
+- Indexes related to `PRIMARY KEY` and `UNIQUE` constraints
+- Unsupported system `GRANT` privileges
+
+  EDB supports granting only `CREATE DATABASE LINK`, `CREATE PUBLIC DATABASE LINK`, `DROP PUBLIC DATABASE LINK`, and `EXEMPT ACCESS POLICY` system privileges. Other `GRANT` statements aren't supported and are removed from the DDL file. 
```

Comment:
It looked like the word "are" was missing from the original. Is my edit correct?

Reply:
You are correct, good catch!